docs/gettingstarted/index.rst

```diff
@@ -84,6 +84,14 @@ Getting started with Flocker
          .. task:: test_homebrew flocker-|latest-installable|
             :prompt: you@laptop:~$
 
+         Ubuntu 15.04
```

Comment:
My understanding is that as that isn't a blocker to FLOC-2008 it might not be in 1.0. Perhaps @rogthedodge can clarify. If that's the case let's not make this worse.